src/main/java/uk/gov/hmcts/reform/blobrouter/services/RejectedBlobChecker.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.hmcts.reform.blobrouter.services;
+
+import com.azure.storage.blob.models.BlobItem;
+import org.apache.commons.lang3.Validate;
+
+import java.time.Duration;
+
+import static java.time.OffsetDateTime.now;
+
+public class RejectedBlobChecker {
```

Comment:
Should this be a service?

Reply:
You mean the annotation?
Not yet. No config in app yaml etc.